apps/base/templates/base.html

```diff
@@ -41,6 +42,10 @@
                     <a class="navbar-child navbar-user" href="{% url 'moodytunes:playlists' %}">Playlists</a>
                     <a class="navbar-child navbar-user" href="{% url 'moodytunes:browse' %}">Browse</a>
                     <a class="navbar-child navbar-user" href="{% url 'moodytunes:suggest' %}">Suggest Song</a>
+
+                    {% flag 'spotify_auth' %}
+                        <a class="navbar-child navbar-user" href="{% url 'moodytunes:spotify-auth' %}">Authenticate With Spotify</a>
```

Comment:
Instead of having this a link in the navbar, we can redirect a user to this page if they don't have a `SpotifyUserAuth` record when trying to export a playlist. We can delay this integration for now

Reply:
Leave in for now, it's useful for testing purposes. We'll revisit this when its time to go live